src/main/java/xyz/nucleoid/plasmid/game/ManagedGameSpace.java

```diff
@@ -270,7 +272,9 @@ private void onRemovePlayer(ServerPlayerEntity player) {
     public CompletableFuture<StartResult> requestStart() {
         return Scheduler.INSTANCE.submit(server -> {
             try {
-                return this.invoker(RequestStartListener.EVENT).requestStart();
+                StartResult startResult = this.invoker(RequestStartListener.EVENT).requestStart();
+                GameEvents.START_REQUEST.invoker().onRequestStart(this, startResult);
```

Comment:
The result from this looks like it is never used?

Reply:
Fixed!